js_modules/dagit/packages/core/src/runs/LogsRowStructuredContent.tsx

```diff
@@ -144,6 +144,8 @@ export const LogsRowStructuredContent: React.FC<IStructuredContentProps> = ({nod
           timestamp={node.timestamp}
         />
       );
+    case 'AssetMaterializationPlannedEvent':
+      return <DefaultContent eventType={eventType} message={node.message} />;
```

Comment:
This should never get evaluated, right?  Would it make sense to just have this return null instead of putting the logic in `filterLogs`?

Reply:
If this returned null, the row would still partially display with the timestamp:
![image](https://user-images.githubusercontent.com/29110579/163066711-98950f03-bce7-47aa-8649-d33184ff117d.png)